lms/startup.py

```diff
@@ -41,6 +41,8 @@ def run():
 
     add_mimetypes()
 
+    microsite.enable_microsites(log)
```

Comment:
@felipemontoya I'm not able to understand the usage of this statement. Why we need to do the same thing once before `django.setup()` and once afterwards? Could you please explain a little?